### ISSUE TITLE
Update support

### ISSUE THREAD

**Issue by xAffan** [chiteroman/TrickyStore#1]
Since you are going to be focusing on tricky store instead of PiF, please add module update support.

**Comment by VisionR1** [comment]
> Since you are going to be focusing on tricky store instead of PiF, please add module update support.

If you mean support for Tricky Store, already a thread on XDA.
https://xdaforums.com/t/tricky-store-the-next-generation-pif.4683446/

**Comment by xAffan** [comment]
> > Since you are going to be focusing on tricky store instead of PiF, please add module update support.
> 
> If you mean support for Tricky Store, already a thread on XDA. https://xdaforums.com/t/tricky-store-the-next-generation-pif.4683446/

No. I mean update support within magisk whenever there's a new release.

**Comment by VisionR1** [comment]
That's make sense l.
